projects/common/src/time/time-duration.ts

```diff
@@ -12,6 +14,10 @@ export class TimeDuration {
     return this.millis;
   }
 
+  public static parse(durationString: string): TimeDuration {
+    return new TimeDuration(toSeconds(parse(durationString)), TimeUnit.Second);
+  }
+
```

Comment:
can we also update `toIso8601DurationString` to now use library methods?

Reply:
@anandtiwary the library does not have a method to convert to a duration string :)